packages/admin/src/routes/entities/components/ActionView/ActionView.js

```diff
@@ -0,0 +1,49 @@
+import React, {useEffect} from 'react'
+import PropTypes from 'prop-types'
+import queryString from 'query-string'
+import {intlShape} from 'react-intl'
+
+import Action from '../Action'
+
+const ActionView = ({location, setCurrentViewTitle, currentViewInfo, intl}) => {
+  useEffect(() => {
+    if (currentViewInfo) {
+      setCurrentViewTitle(intl.formatMessage({id: `client.actions.${currentViewInfo.actionId}.title`}))
+    }
+
+    return () => {
+      setCurrentViewTitle(null)
+    }
+  }, [currentViewInfo])
+
+  if (!currentViewInfo) {
+    return null
+  }
+
+  // TODO: Action better query param handler. allow tql
```

Comment:
what about this todo? left here intentionally?

Reply:
i thought you will work on this. I'll remove the comment anyway